README.md

```diff
@@ -1,89 +1,114 @@
-- [Code guidelines](#code-guidelines)
+Cloudbeat analyzes cloud assets for security compliance and sends findings to Elasticsearch as part of
+the [Cloud Security Posture](https://www.elastic.co/blog/secure-your-cloud-with-elastic-security) plugin in Kibana.
+
+## Getting Started
+
+To get started with Cloud Security Posture on your cluster, see
+our [documentation](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-unmanaged).
+
+- [Setup KSPM for Amazon EKS clusters](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-unmanaged)
+
+- [Setup KSPM for unmanaged Kubernetes clusters](https://www.elastic.co/guide/en/security/master/get-started-with-kspm.html#kspm-setup-eks-start)
+
+___
+
+## Cloudbeat Development
+
+### Table of contents
 
+- [Prerequisites](#prerequisites)
+- [Deploying Cloudbeat as a process](#deploying-cloudbeat)
+    - [Unmanaged Kubernetes](#clean-up)
+    - [Amazon Elastic Kubernetes Service (EKS)](#amazon-elastic-kubernetes-service-(EKS))
+- [Deploying Cloudbeat with Elastic-Agent](#running-cloudbeat-with-elastic-agent)
 
 ## Prerequisites
-1. [Hermit by Cashapp](https://cashapp.github.io/hermit/usage/get-started/)
-2. Elasticsearch with the default username & password (`elastic` & `changeme`) running on the default port (`http://localhost:9200`)
-3. Kibana with running on the default port (`http://localhost:5601`)
-4. Install and configure [Elastic-Package](https://github.com/elastic/elastic-package) (you may need to [authenticate](https://docker-auth.elastic.co/github_auth))
-5. Set up the local env:
 
-- Install & activate hermit
-  ```zsh
-  curl -fsSL https://github.com/cashapp/hermit/releases/download/stable/install.sh | /bin/bash
-  ```
-	```zsh
-  . ./bin/activate-hermit
-  ```
-- Run setup env recipe
+1. We use [Hermit](https://cashapp.github.io/hermit/usage/get-started/) to keep all our tooling in check. Install it
+   with the following command:
+    ```zsh
+    curl -fsSL https://github.com/cashapp/hermit/releases/download/stable/install.sh | /bin/bash
+    . ./bin/activate-hermit
+    ```
+
+   > **Note**
+   This will download and install hermit into `~/bin`. You should add this to your `$PATH` if it isn't already.
+
+   It is also recommended to add hermit's [shell integration](https://cashapp.github.io/hermit/usage/shell/)
+
+2. Elastic stack running locally, preferably using [Elastic-Package](https://github.com/elastic/elastic-package) (you
+   may need to [authenticate](https://docker-auth.elastic.co/github_auth))
+   For example, spinning up 8.5.0 stack locally:
+
+    ```zsh
+    eval "$(elastic-package stack shellinit)" # load stack environment variables
+    elastic-package stack up --version 8.5.0 -v -d
+    ```
+
+- _optional:_ Create local kind cluster to test against
   ```zsh
-  just setup-env
+  just create-kind-cluster
+  just elastic-stack-connect-kind # connect it to local elastic stack
   ```
 
+# Deploying Cloudbeat
```

Comment:
`h1` usually only shows up once at the top. i would change this to `h2`, same as `Cloudbeat Development`

Reply:
nit: 
`Cloudbeat Development` -> `Development`
`Deploying Cloudbeat` -> `Deployment`